alembic/env.py

```diff
@@ -20,17 +20,16 @@
 # for 'autogenerate' support
 # from myapp import mymodel
 # target_metadata = mymodel.Base.metadata
-import mappings
-target_metadata = mappings.Base.metadata
+import db.models as models
```

Comment:
`from db import models` has the same effect and is a bit cleaner imo. `as` is usually used to abbreviate and prevent name conflicts. in this case, we just want to import the models submodule from db without actually renaming it.